app/locations/trailing-history.js

```diff
@@ -0,0 +1,13 @@
+import HistoryLocation from '@ember/routing/history-location';
+
+export default class TrailingHistoryLocation extends HistoryLocation {
+  formatURL() {
+    let url = super.formatURL(...arguments);
```

Comment:
What is the type of `url`, returned from `formatURL`? Is it guaranteed to be a string?

Reply:
This will always be a string for sure, unless we have a serious problem in the whole use of URLs in every ember app ever made 😂